src/asmcnc/apps/maintenance_app/screen_maintenance.py

```diff
@@ -602,8 +639,13 @@ def on_enter(self):
             self.tab_panel.switch_to(self.laser_tab)
         elif self.landing_tab == 'spindle_tab':
             self.tab_panel.switch_to(self.spindle_tab)
+        elif self.landing_tab and self.m.theateam() == 'spindle_health_check_tab':
```

Comment:
I think you got this the wrong way around

Reply:
oh dear 😅


but somehow it seemed to work ?? 🤯

Reply:
ah no, I just don't remember the order in which I tested and added, and then didn't catch it on retesting 🤦‍♀️